main.cpp

```diff
@@ -13,75 +13,486 @@ What you create in this project will be used as the basis of Project 5 in the co
+    4) Amount of money to ride (float)
+    5) Peak Number of riders a day (int)
+3 things it can do:
+    1) stop
+    2) slow down
+    3) speed up
+ */
+struct Rollercoaster
+{
+    //Amount of seats (int)
+    int amountOfSeats = 40;
+    //number of max speed mph (int)
+    int maxSpeedMph = 80;
+    //Average number of laps a day (int)
+    int averageLaps = 30;
+    //Amount of money to ride (float)
+    float cost = 2.99f;
+    //Peak Number of riders a day (int)
+    int ridersPerDayInPeakTime = 200;
+    //stop
+    void stop ();
+    //slow down
+    void slowDown ();
+    //speed up
+    void speedUp ();
+};
+/*
+Thing 4) Art Musuem
+5 properties:
+    1) Number of works of art (int)
+    2) Number of statues (int)
+    3) Most iconic piece (std::string" "")
+    4) Number of visitors (int)
+    5) Number of new pieces (int)
+3 things it can do:
+    1)Add new pieces
+    2)Remove old pieces
+    3)Restore ancient art */
+
+struct artMusuem
+{
+//Number of works of art (int)
+int worksOfArt = 90;
+//Number of statues (int)
+int statues = 20;
+// Most iconic piece (std::string" "")
+std::string mostFamous = "TheMonaLisa";
+//) Number of visitors (int)
+int vistorsPerYear = 19000;
+//Number of new pieces (int)
+int NumberOfNewPieces = 20;
+//Add new pieces
+void addNewArt (bool statue = true, bool painting = false);
+//Remove old pieces
+void removeOld ();
+//Restore ancient art
+void restoreArt ();
+};
+
+/*
+Thing 5) Mixing Desk
+    5 properties:
+        1) Name of brand (std::string)
+        2) Number of Input Channels (int)
+        3) Number of Output Channels (int)
+        4) Does have a Mix bus Compressor (bool)
+        5) Amount of money (int)
+    3 things it can do:
+        1)Turn faders to 0
+        2)Mix bus compression
+        3)Initiate recording
+    */
+struct mixingDesk
+{
+//Name of brand 
+std::string brand = "AWS";
+//Number of Input Channels 
+int inputChannels = 24;
+//Number of Output Channels 
+int outputChannels = 24;
+//Does have a Mix bus Compressor 
+bool hasMixBusCompressor = true;
+//Amount of money 
+int cost = 70000;
+//Turn faders to 0
+void Faders0();
+//Mix bus compression
+bool mixBusOn ();
+//Initiate recording
+};
+    /*
+    Thing 6)
+    5 properties:Imac
+        1)Amount of Storage in GB (int)
+        2)Number of pixels in display (int)
+        3)Amount of GPU power in GHz (float)
+        4)Amount of RAM (int)
+        5)Model Number name (std::string)
+    3 things it can do:
+        1)Record inputs to the mixing desk 
+        2)Store the recordings
+        3)Take pictures
+    */
+struct Imac
+{
+//Amount of Storage in GB 
+int StorageInGB = 256;
+//Number of pixels in display
+int pixels = 4056;
+//Amount of GPU power in GHz
+int GpuInGHz = 8;
+//Amount of RAM 
+int RamInGB = 2;
+//Model Number name
+std::string modelName = "TY34550B";
+//Record inputs in to the mixing desk 
+void record ();
+//Store the recordings
+void save ();
+//Take pictures
+void takePicture ();
+};
```

Comment:
- fix this indentation
- variable names and function names need to start with a lower-case letter
- [ ] revise